dra-evolution/testdata/pools-two-nodes-dgxa100.yaml

```diff
@@ -0,0 +1,7695 @@
+      - capacity: "1"
+        name: gpu-0-memory-slices-2
+      - capacity: "1"
+        name: gpu-0-memory-slices-3
+    - attributes:
+      - name: mig-profile
+        string: 3g.20gb
+      name: gpu-0-mig-3g.20gb-4
+      sharedCapacityConsumed:
+      - capacity: "42"
+        name: gpu-0-multiprocessors
+      - capacity: "3"
+        name: gpu-0-copy-engines
+      - capacity: "2"
+        name: gpu-0-decoders
+      - capacity: 19968Mi
+        name: gpu-0-memory
+      - capacity: "1"
+        name: gpu-0-memory-slices-4
+      - capacity: "1"
+        name: gpu-0-memory-slices-5
+      - capacity: "1"
+        name: gpu-0-memory-slices-6
+      - capacity: "1"
+        name: gpu-0-memory-slices-7
+    - attributes:
+      - name: mig-profile
+        string: 4g.20gb
+      name: gpu-0-mig-4g.20gb-0
+      sharedCapacityConsumed:
+      - capacity: "56"
+        name: gpu-0-multiprocessors
+      - capacity: "4"
+        name: gpu-0-copy-engines
+      - capacity: "2"
+        name: gpu-0-decoders
+      - capacity: 19968Mi
+        name: gpu-0-memory
+      - capacity: "1"
+        name: gpu-0-memory-slices-0
+      - capacity: "1"
+        name: gpu-0-memory-slices-1
+      - capacity: "1"
+        name: gpu-0-memory-slices-2
+      - capacity: "1"
+        name: gpu-0-memory-slices-3
+    - attributes:
+      - name: mig-profile
+        string: 7g.40gb
+      name: gpu-0-mig-7g.40gb-0
+      sharedCapacityConsumed:
+      - capacity: "98"
+        name: gpu-0-multiprocessors
+      - capacity: "7"
+        name: gpu-0-copy-engines
+      - capacity: "5"
+        name: gpu-0-decoders
+      - capacity: "1"
+        name: gpu-0-jpeg-engines
+      - capacity: "1"
+        name: gpu-0-ofa-engines
+      - capacity: 40192Mi
+        name: gpu-0-memory
+      - capacity: "1"
+        name: gpu-0-memory-slices-0
+      - capacity: "1"
+        name: gpu-0-memory-slices-1
+      - capacity: "1"
+        name: gpu-0-memory-slices-2
+      - capacity: "1"
+        name: gpu-0-memory-slices-3
+      - capacity: "1"
+        name: gpu-0-memory-slices-4
+      - capacity: "1"
+        name: gpu-0-memory-slices-5
+      - capacity: "1"
+        name: gpu-0-memory-slices-6
+      - capacity: "1"
+        name: gpu-0-memory-slices-7
+    - attributes:
+      - name: mig-profile
+        string: 1g.5gb+me
+      name: gpu-0-mig-1g.5gb-me-0
+      sharedCapacityConsumed:
+      - capacity: "14"
+        name: gpu-0-multiprocessors
+      - capacity: "1"
+        name: gpu-0-copy-engines
+      - capacity: "1"
+        name: gpu-0-decoders
+      - capacity: "1"
+        name: gpu-0-jpeg-engines
+      - capacity: "1"
+        name: gpu-0-ofa-engines
+      - capacity: 4864Mi
+        name: gpu-0-memory
+      - capacity: "1"
+        name: gpu-0-memory-slices-0
+    - attributes:
+      - name: mig-profile
+        string: 1g.5gb+me
+      name: gpu-0-mig-1g.5gb-me-1
+      sharedCapacityConsumed:
+      - capacity: "14"
```

Comment:
"capacity" is a producer term, not a consumer term.  This should be "quantity" or "amount" or "request" or something.

Reply:
It reuses the same struct as that used to declare the capacity. We would need a new struct then (which seems fine).